incubator/stackdriver-metrics-adapter/values.yaml

```diff
@@ -0,0 +1,23 @@
+replicaCount: 1
```

Comment:
Can this run with multiple replicas?  I imagine people don't want to lose their custom metrics when an update happens.

Reply:
yep it can work with 2 replicas